src/screens/TopicPageScreen/elements/BottomSheetFollow.tsx

```diff
@@ -54,12 +52,7 @@ const BottomSheetFollow = forwardRef((props: BottomSheetFollowProps, ref: Ref<RB
       setIsFollow(true);
       try {
         await followTopic(topicName, isIncognito);
-        getTopicDetail(topicName);
-        const discoveryInitialTopicResponse = await DiscoveryRepo.fetchInitialDiscoveryTopics();
-        DiscoveryAction.setDiscoveryInitialTopics(
-          discoveryInitialTopicResponse.suggestedTopics as any,
-          discoveryDispatch
-        );
+        updateFollowTopicDiscoveryContext(true, {topic_id: topicId}, true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateFollowTopicDiscoveryContext` function is being called with `true` as the first argument, `{topic_id: topicId}` as the second argument, and `true` as the third argument. It's not clear what these arguments represent without looking at the function definition. For better readability, consider using an options object as the argument so that the properties can be named.

```diff
- updateFollowTopicDiscoveryContext(true, {topic_id: topicId}, true);
+ updateFollowTopicDiscoveryContext({isFollowing: true, topic: {id: topicId}, shouldUpdate: true});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->